tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/OdsApiSteps.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Steps.Common;
+using NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Utils;
+using RestEase;
+using TechTalk.SpecFlow;
+using WireMock.Admin.Mappings;
+using WireMock.Client;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.IntegrationTests.Steps
+{
+    [Binding]
+    public sealed class OdsApiSteps
+    {
+        private readonly ScenarioContext _context;
+            private readonly Settings _settings;
+
+        public OdsApiSteps(ScenarioContext context, Settings settings)
+        {
+            _context = context;
+            _settings = settings;
+        }
+
+        [Given(@"Ods API is down")]
+        public async Task CallToTheOdsApiWillFail()
+        {
+            await ClearMappings();
+            var model = CreateMappingModel("/ORD/2-0-0/organisations/*", 500);
+
+            await AddMapping(model);
```

Comment:
I'm not sure the `model` variable adds much value here. We could just do

```c#
await ClearMappings();
await AddMapping(CreateMappingModel("/ORD/2-0-0/organisations/*", 500));
```

Reply:
👍